app-backend/batch/src/main/scala/stacExport/v2/CampaignExport.scala

```diff
@@ -264,7 +264,11 @@ case class ExportData private (
                 file,
                 s"images/${sceneItem.value.id}/item.json"
               )
-          case _ => IO.pure(())
+          case _ =>
+            encodableToFile(
+              (withCollection `compose` withParentLinks)(sceneItem.value),
+              file,
+              s"images/${sceneItem.value.id}/item.json")
```

Comment:
basically we uh still wanna write the file out even if we don't want to write out a COG first 🤦🏻‍♂️ feel like I should have caught this in previous reviews